types/test.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
```

Comment:
let's just nuke the tests since they are causing problems. Maybe we can write an example later using typescript instead?

Reply:
`dtslint@0.6.0` has been released since I wrote this test, but doesn't fix the errors ☹️

I've optimistically left the file there, comment out with how to reenable

Reply:
Looking at the source for `dtslint`, I disabled the [dependencies checking](https://github.com/Microsoft/dtslint/blob/4b2fc0746e4606a3ced8c35e55bc96f861088b0e/src/lint.ts#L25-L28) and the test we've got here passes with only a couple changes. Disappointing, but shows we are on the right path and don't think we should abandon the existing test so eagerly

Reply:
I think I fixed the test by adding TypeScript version (3.0) to `index.d.ts`